runtime/doc/quickfix.txt

```diff
@@ -1921,6 +1922,63 @@ error messages into a format that quickfix mode will understand.  See the
 start of the file about how to use it.  (This script is deprecated, see
 |compiler-perl|.)
 
-
+=============================================================================
+10. Customizing the quickfix window		*quickfix-window-function*
+
+The default format for the lines displayed in the quickfix window and location
+list window is:
+
+    <filename>|<lnum> col <n>|<text>
+
+This can be customized by setting the 'quickfixtextfunc' option to a Vim
+function.  This function will be called with a dict argument for every entry
+in the quickfix list. The dict argument will have the following fields:
+
+    quickfix	set to 1 when called for a quickfix list
```

Comment:
what else would it be set to ? Presumably 0 for location list ?

Reply:
Yes. I will update the help text.